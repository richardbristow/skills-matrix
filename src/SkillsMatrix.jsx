import React from 'react';
import { Container } from 'react-bootstrap';

const SkillsMatrix = () => (
  <Container>
    <h1>Skills Matrix</h1>
  </Container>
);

export default SkillsMatrix;
