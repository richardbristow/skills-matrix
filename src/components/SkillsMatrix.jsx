import React from 'react';
import TopNavBar from './TopNavBar';

const SkillsMatrix = () => <TopNavBar />;

export default SkillsMatrix;
