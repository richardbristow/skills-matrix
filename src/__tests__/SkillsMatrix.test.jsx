import React from 'react';
import { shallow } from 'enzyme';
import SkillsMatrix from '../SkillsMatrix';

describe('SkillsMatrix', () => {
  it('renders without throwing errors', () => {
    const wrapper = shallow(<SkillsMatrix />);
    expect(wrapper.exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
