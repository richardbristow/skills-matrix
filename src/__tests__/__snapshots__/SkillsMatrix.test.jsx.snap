// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkillsMatrix renders without throwing errors 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SkillsMatrix />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Skills Matrix
      </h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Skills Matrix",
      },
      "ref": null,
      "rendered": "Skills Matrix",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Skills Matrix
        </h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Skills Matrix",
        },
        "ref": null,
        "rendered": "Skills Matrix",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
